.github/workflows/verify.yaml

```
@@ -34,7 +34,7 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@4afd733a84b1f43292c63897423277bb7f4313a9 # v8.0.0
         with:
-          version: v2.2.1
+          version: v2.6.1
```

i would drop just to 


```suggestion
          version: v2.6
```

so that will get the lastest version always and reduce the need for us to bump everytime

---

Done